### DISCUSSION TITLE
How would having a low IQ affect me in the real world and in coding? (Yes, I am that dumb)

### DISCUSSION THREAD

**Discussion by ghost** [community/community#114297] [Programming Help] [answered]
### Body

# How would having a low IQ affect me in the real world and in coding? (Yes, I am that dumb)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ghost** [comment]
Should I give up on coding?

**Reply by moustafa-elmaghraby** [reply]
No bro just keep it up and try harder as long as u know the problem u will get the solution and there is no one with low IQ if u love coding u will give it every min of ur time and in return u will be so good day by day god bless u and have a nice day

**Comment by Kakar21** [comment]


**Reply by ghost** [reply]
I already learned HTML, CSS, JavaScript, Python, Assembly and C++

**Reply by ghost** [reply]
I wouldn't say I master, but I've used them. The language I am closest to mastering is HTML, but yes, I've learned them.

**Reply by ghost** [reply]
Myself but I copied online until I could remember.

**Reply by ghost** [reply]
I am a frontend developer; I can show you some of my websites if you like.

**Reply by ghost** [reply, marked as answer]
https://github.com/PyDOS8/Space-Invaders-Remake-Website This is just pure JavaScript as of now.
https://github.com/PyDOS8?tab=repositories&q=&type=&language=html&sort=
 https://github.com/PyDOS8/Space-Invaders-Remake. I written the Assembly code for this one not the Java, but I cannot take coding and Mr. Brunetti said you needed to be smart to take it, so I never took a course I learned all online.

**Reply by ghost** [reply]
It is nothing like Python.
